.prettierignore

```diff
@@ -1,7 +1,3 @@
-# Ignore everything except the src folder:
-/*
-!/src
```

Comment:
we said we do ONLY prettify src agreed in the team. Please revert

Reply:
done

Reply:
done.